### PR TITLE
https://issues.redhat.com/browse/ACM-11181 Partial matching search

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: observability/search_intro.adoc [hunk 1]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `!` or `!=`
+| string, number
+| This represents the _NOT_ operation, which means to exclude from the search results.
+
+| `<`, `<=`, `>`, `>=`
+| number
+| 
+
+| `TODO`
```
Comment (SherinV) [inline observability/search_intro.adoc]: For dates, the default operation is ">".

File: observability/search_intro.adoc [hunk 2]
```diff
@@ -170,6 +171,36 @@ spec:
...
 ----
 
+[#search-operations]
+== Search operations and data types
+
+You can search with conditions by using characters, such as `+>, >=, <, <=, !=+`. See the following search operation table:
+
+.Search operation table
+|===
+| Operation | Data type | Description
```
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| Default operation | Data type | Description
```

File: observability/search_intro.adoc (hunk 1 above)
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| `>`
```

File: observability/search_intro.adoc [hunk 3]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `=`
+| string, number
+| This is the default operation.
+
+| `!` or `!=`
+| string, number
+| This represents the _NOT_ operation, which means to exclude from the search results.
+
+| `<`, `<=`, `>`, `>=`
```
Comment (SherinV) [inline observability/search_intro.adoc]: Not sure why the <= shows up weird in the rich preview. Everything else, looks good.

**Review by SherinV — APPROVED** [review]

File: observability/search_intro.adoc [hunk 4]
```diff
@@ -170,6 +171,36 @@ spec:
   operator: Exists
 ----
 
+[#search-operations]
+== Search operations and data types
+
+You can search with conditions by using characters, such as `+>, >=, <, <=, !=+`. See the following search operation table:
```
Comment (jc-berger) [inline observability/search_intro.adoc]: What is the benefit or purpose of users searching with conditions? See my suggested rewrite, and make changes if I am misunderstanding the benefit. 

```suggestion
To identify what operations or data types you need to implement or modify, use conditions to find these operations or data types. You can search with conditions by using characters, such as `+>, >=, <, <=, !=+`. See the following search operation table:
```

**Review by jc-berger — APPROVED** [review]
/lgtm

File: observability/search_intro.adoc (hunk 4 above)
Comment (dockerymick) [inline observability/search_intro.adoc]: @jc-berger thanks Jake, how about this? 
```suggestion
Specify your search query by using search operations as conditions. Characters such as `+>, >=, <, <=, !=+` are supported. See the following search operation table:
```

File: observability/search_intro.adoc [hunk 5]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `=`
+| string, number
+| This is the default operation.
+
+| `!` or `!=`
+| string, number
+| This represents the _NOT_ operation, which means to exclude from the search results.
+
+| `<`, `<=`, `>`, `>=`
```
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| `<`, `\<=`, `>`, `>=`
```

File: observability/search_intro.adoc [hunk 6]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `=`
+| string, number
+| This is the default operation.
+
+| `!` or `!=`
+| string, number
+| This represents the _NOT_ operation, which means to exclude from the search results.
+
+| `<`, `\<=`, `>`, `>=`
```
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| `<, <=, >, >=`
```

File: observability/search_intro.adoc [hunk 7]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `<, <=, >, >=`
+| number
+| 
+
+| `>`
+| date
+| Dates matching the last hour, day, week, month, and year.
+
+| `*`
```
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| `\*` 
```

File: observability/manage_search.adoc [hunk 8]
```diff
@@ -98,7 +98,6 @@ See the following examples that you can select for filtering: `name`, `namespace
 +
 *Notes:*
 +
-** You can also search with conditions by using characters, such as `+>, >=, <, <=, !=+`.
 ** When you search for more than one property selector with multiple values, the search returns either of the values that were queried. View the following examples:
 ** When you search for `kind:pod name:a`, any pod named `a` is returned.
```
Comment (SherinV) [inline observability/manage_search.adoc]:
```suggestion
** When you search for `kind:Pod name:a`, any pod named `a` is returned.
```

File: observability/manage_search.adoc [hunk 9]
```diff
@@ -98,7 +98,6 @@ See the following examples that you can select for filtering: `name`, `namespace
...
 ** When you search for `kind:pod name:a,b`, any pod named `a` or `b` are returned.
```
Comment (SherinV) [inline observability/manage_search.adoc]:
```suggestion
** When you search for `kind:Pod name:a,b`, any pod named `a` or `b` are returned.
```

Comment (SherinV) [reply observability/manage_search.adoc]: Please capitalize 'Pod' in the examples below. Thanks.

File: observability/search_intro.adoc [hunk 10]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `<, <=, >, >=`
+| number
+| 
+
+| `>`
+| date
+| Dates matching the last hour, day, week, month, and year.
+
+| `\*`
```
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| \`*` 
```

File: observability/search_intro.adoc [hunk 11]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `<, <=, >, >=`
+| number
+| 
+
+| `>`
+| date
+| Dates matching the last hour, day, week, month, and year.
+
+| \`*`
```
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| `*`\ 
```

File: observability/search_intro.adoc [hunk 12]
```diff
@@ -170,6 +171,36 @@ spec:
...
+
+| `<, <=, >, >=`
+| number
+| 
+
+| `>`
+| date
+| Dates matching the last hour, day, week, month, and year.
+
+| `*`\
```
Comment (dockerymick) [inline observability/search_intro.adoc]:
```suggestion
| `*`
```

**Comment by dockerymick** [comment]
Accepted the suggestions from Sherin. Thanks for the reviews and approvals. Merging this

**Pull request merged**